lib/routes/account.js

```diff
@@ -972,6 +979,7 @@ module.exports = function (
       },
       handler: function (request, reply) {
         log.begin('Account.devices', request)
+        var language = request.query.language || 'en'
```

Comment:
Yeah, using language as provided by content-server is probably safer, less likely to result in weird mismatches